frontend/app/components/redesign/components/InputField.tsx

```diff
@@ -43,8 +59,17 @@ export const InputField = forwardRef<HTMLInputElement, InputFieldProps>(
           {...props}
         />
         {error && <p className="text-xs text-text-error">{displayError}</p>}
-        {helpText && !error && (
-          <p className="text-xs text-text-secondary">{helpText}</p>
+        {(helpText || showCounter) && !error && (
+          <div className="flex justify-between gap-xs">
```

Comment:
If both the text have same styles (`text-xs text-text-secondary`), we can add the styles to parent div.